project_template/package.json

```diff
@@ -48,7 +48,7 @@
     "babel-eslint": "^7.2.3",
     "babel-loader": "^8.0.0",
     "chai": "^3.5.0",
-    "chromedriver": "^2.35.0",
+    "chromedriver": "^75.0.0",
```

Comment:
Latest version is `76.0.0`. Might as well use that?

Reply:
Agree.

Reply:
Changed it.